src/app/navigation/Home.screen.tsx

```diff
@@ -8,30 +15,35 @@ export type HomeRouteParams = undefined;
 
 export const Home = () => {
   const [movies, setMovies] = useState([]);
```

Comment:
maybe we can type the Array
`
const [movies, setMovies] = useState<Array<Movie>>([]);
`